src/main/java/studio/magemonkey/fusion/cfg/PConfigManager.java

```diff
@@ -22,4 +22,8 @@ public static PlayerConfig getPlayerConfig(Player player) {
 
         return pConfig.get(player.getUniqueId());
     }
+
+    public static void resetPlayerConfigs() {
```

Comment:
'Reset' here feels a little aggressive and like we are resetting their data, not just unloading the configs from the list. Maybe `clearPConfigCache` or something?

Reply:
I agree with you, I have renamed the method. I saw that its not even used, but I would still keep it in case we need it someday. What do you think?

Reply:
Yeah, we can go ahead and keep it in case we need it